assets/src/edit-story/components/form/dateTime/datePicker.js

```diff
@@ -1,101 +1,49 @@
-// eslint-disable-next-line eslint-comments/disable-enable-pair
-/* eslint-disable header/header */
-/* Disable reason: This file is based on Gutenberg Datepicker */
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
 /**
  * External dependencies
  */
-import moment from 'moment';
-// Needed to initialise the default datepicker styles.
-// See: https://github.com/airbnb/react-dates#initialize
-import 'react-dates/initialize';
-import { DayPickerSingleDateController } from 'react-dates';
-import 'react-dates/lib/css/_datepicker.css';
+import Calendar from 'react-calendar';
+import 'react-calendar/dist/Calendar.css';
```

Comment:
/to @swissspidy, I've been trying to clarify this for a bit: does our WebPack configuration supports direct CSS imports? Can you point me to where this is configurable?

Reply:
Yes it does. We're using [`MiniCssExtractPlugin`](https://webpack.js.org/plugins/mini-css-extract-plugin/) for this, as well as [`OptimizeCSSAssetsPlugin`](https://www.npmjs.com/package/optimize-css-assets-webpack-plugin) to optimize the extracted CSS.

Any CSS file that is imported like this within `assets/src/edit-story` will be extracted and combined into `assets/css/edit-story.css`.

The relevant webpack configuration is here:

https://github.com/google/web-stories-wp/blob/6237ba14457e84c4457568f804a40130eade57e8/webpack.config.js#L87-L100

Reply:
Thanks! This is very helpful